homeassistant/components/deconz/device_trigger.py

```diff
@@ -189,9 +196,15 @@
     (CONF_DOUBLE_TAP, CONF_SIDE_4): 4004,
     (CONF_DOUBLE_TAP, CONF_SIDE_5): 1001,
     (CONF_DOUBLE_TAP, CONF_SIDE_6): 5005,
-    (CONF_AWAKE, ""): 7000,
-    (CONF_FREE_FALL, ""): 7008,
-    (CONF_SHAKE, ""): 7007,
+    (CONF_AWAKE, ""): {CONF_GESTURE: 0},
+    (CONF_SHAKE, ""): {CONF_GESTURE: 1},
+    (CONF_FREE_FALL, ""): {CONF_GESTURE: 2},
+    (CONF_FLIP_90, ""): {CONF_GESTURE: 3},
+    (CONF_FLIP_180, ""): {CONF_GESTURE: 4},
+    (CONF_MOVE_ANY, ""): {CONF_GESTURE: 5},
+    (CONF_DOUBLE_TAP_ANY, ""): {CONF_GESTURE: 6},
+    (CONF_TURN_CW, ""): {CONF_GESTURE: 7},
+    (CONF_TURN_CCW, ""): {CONF_GESTURE: 8},
```

Comment:
Make all triggers of cube be either {CONF_EVENT: x} or {CONF_GESTURE: x}

Reply:
Does it make sense to also make the triggers of all the other remotes {CONF_EVENT: x}? Than the check below can be completely removed.

Reply:
Sure. Let's keep it consistent